src/unix/fs.c

```diff
@@ -1240,9 +1241,28 @@ static ssize_t uv__fs_write(uv_fs_t* req) {
   return r;
 }
 
-static ssize_t uv__fs_copyfile(uv_fs_t* req) {
 #if defined(__APPLE__) && !TARGET_OS_IPHONE
-  /* On macOS, use the native copyfile(3). */
+
+static ssize_t uv__fs_clonefile_mac(uv_fs_t* req) {
+  static int has_clone_acl = 1;
```

Comment:
```suggestion
  static _Atomic int has_clone_acl = 1;
```
And use `atomic_load_explicit()` (and store) with `memory_order_relaxed`.

Ideally, set it to zero initially, because then it ends up in the .bss instead of the .data section. That's why libuv generally uses `no_foo` instead of `have_foo`.

Reply:
Other static vars don't use `_Atomic` and use `uv__load_relaxed` and `uv__store_relaxed`.